setup.cfg

```diff
@@ -53,7 +53,7 @@ filterwarnings =
     error
     # Suppress deprecation warnings in other packages
     ignore:lib2to3 package is deprecated::scspell
-    ignore:pkg_resources is deprecated as an API::flake8_import_order
+    ignore:pkg_resources is deprecated as an API::
```

Comment:
This needed to be loosened because we're actually running the code in the tests now. I'm going to follow up with a change that will break our dependency on `pkg_resources` directly, at which point I'll revert this hunk.